.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: groovy
+
+sudo: required
+
+services:
+    - docker
+
+before_install:
+    - make build
+    - make run
+
+script:
```

Comment:
let's put our usual stanza in there so that pushes only build once.
```
branches:
    only:
      - master
```
Reference: https://openedx.atlassian.net/wiki/display/TE/Troubleshooting+Travis+builds